adaptation_layer/driver/manager.py

```diff
@@ -30,6 +31,8 @@ def get_driver(nfvo_id: int, nfvo_type: str, nfvo_cred: Dict) -> Driver:
         _drivers[nfvo_id] = OSM(nfvo_cred)
     elif nfvo_type == 'onap':
         _drivers[nfvo_id] = ONAP(nfvo_cred)
+    elif nfvo_type == 'cloudify':
```

Comment:
maybe using the cloudify type is not a good idea, cloudify could be very generic, I would prefer to use EVER, specific to the EVER solution

Reply:
ok